collectors/htlcs_collector.go

```diff
@@ -28,6 +40,10 @@ type htlcMonitor struct {
 	// forwards through the node.
 	failureCounter prometheus.Counter
 
+	// failureReasonCounterVec holds counts of our different failure reasons
```

Comment:
Why have two counters (one with a label and one without)? Seems we can get y with only a single counter, and we'd only read the label if we needed to group by that reason or dig down more into what's going on.

Reply:
Indeed, folded this in with the existing countervec.